src/SDKs/SqlManagement/Sql.Tests/DataMaskingScenarioTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using Microsoft.Azure.Management.Sql.Models;
+using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Azure.Management.Sql;
+using Xunit;
+using Microsoft.Azure.Test.HttpRecorder;
+
+namespace Sql.Tests
+{
+    public class DataMaskingScenarioTests
+    {
+        [Fact]
+        public void TestCreateUpdateGetDataMaskingRules()
+        {
+            string testPrefix = "sqlcrudtest-";
+            string suiteName = this.GetType().FullName;
+            SqlManagementTestUtilities.RunTestInNewV12Server(suiteName, "TestCreateUpdateGetDataMaskingRules", testPrefix, (resClient, sqlClient, resourceGroup, server) =>
+            {
+                // Create database
+                //
+                string dbName = SqlManagementTestUtilities.GenerateName(testPrefix);
+                var db1 = sqlClient.Databases.CreateOrUpdate(resourceGroup.Name, server.Name, dbName, new Database()
+                {
+                    Location = server.Location,
+                });
+                Assert.NotNull(db1);
+
+                // Create server firewall rule
+                sqlClient.FirewallRules.CreateOrUpdate(resourceGroup.Name, server.Name, "sqltestrule", new FirewallRule()
+                {
+                    StartIpAddress = "0.0.0.0",
+                    EndIpAddress = "255.255.255.255"
+                });
+
+                // Create test table with columns
+                HttpRecorderMode testMode = HttpMockServer.GetCurrentMode();
+
+                if (testMode != HttpRecorderMode.Playback)
```

Comment:
Please add comment explaining why it's not needed in Playback, i.e. because there is no HTTP requests in this block and it requires the server to really exist